tools/misc/tpm2_eventlog.c

```diff
@@ -27,10 +31,33 @@ static bool on_positional(int argc, char **argv) {
     return true;
 }
 
+static bool on_option(char key, char *value) {
+
+    uint32_t version;
+
+    switch (key) {
+    case 'y':
+        version = (uint32_t)atoi(value);
```

Comment:
use strtoul, because overflow detection, we have a helper for this, use: tpm2_util_string_to_uint32 from tpm2_util.h

Reply:
ah good to know, will use the helper function instead of `atoi`